### ISSUE TITLE
Android 12 support

### ISSUE THREAD

**Issue by negberts** [CrossGeeks/AzurePushNotificationPlugin#65]
Same issue as reported here: 

https://github.com/CrossGeeks/PushNotificationPlugin/issues/175

Is it possible to add Exported = false to the Service tag of  PNMessagingService?

**Comment by eyeveye** [comment]
Yeah, this need to be fixed. Having a hard time to upload to Play store.

**Comment by ghost** [comment]
Also having issues with this.

**Comment by nord-** [comment]
PR: https://github.com/CrossGeeks/AzurePushNotificationPlugin/pull/66

**Comment by EmmanVazz** [comment]
That PR is still not accepted and there is no new version. Is there a work around?

**Comment by EmmanVazz** [comment]
I found a work around. Just take the AndroidManifest.xml service deceleration you find the obj/Debug folder when you make a build and copy it manually into you AndroidManifest.xml in the project with the android:exported flag. Mine looks like this

<service android:name="crc64255f1d4b618a9a90.PNMessagingService" android:exported="false">
      <intent-filter>
        <action android:name="com.google.firebase.MESSAGING_EVENT" />
      </intent-filter>
    </service>

Then when you build again you will see that the AndroidManifest.xml in the obj/Debug has the correct deceleration.

**Comment by GianMaria22** [comment]
> I found a work around. Just take the AndroidManifest.xml service deceleration you find the obj/Debug folder when you make a build and copy it manually into you AndroidManifest.xml in the project with the android:exported flag. Mine looks like this
> 
> Then when you build again you will see that the AndroidManifest.xml in the obj/Debug has the correct deceleration.

Thank you for your suggestion. Maybe I'm doing something wrong because the modified .XML file is always regenerated
